features/product/viewing_products/viewing_product_price_on_products_list.feature

```diff
@@ -0,0 +1,15 @@
+@viewing_products
+Feature: Viewing a product price on products list
+    In order to see products price
+    As a Visitor
+    I want to be able to view a single product price on products list
+
+    Background:
+        Given the store operates on a single channel in "France"
+
+    @todo
+    Scenario: Viewing a products with price on list
+        Given the store has a product "T-shirt watermelon" priced at "€19.00"
+        And this product belongs to "T-Shirt"
+        When I browse products from taxon "T-Shirts"
+        Then I should see the product "T-shirt banana" with price "€19.00"
```

Comment:
I might not be a professional biologist, but i'm quite certain that bananas are not watermelons.